### PR TITLE
feat: add github action for sbom generation

### PR DESCRIPTION
**Pull request by MarcoKorinth**
# Behavior
Every release triggers the github action which uses [Syft](https://github.com/anchore/syft) to generate sbom files in both `SPDX` and `CycloneDX` formats. The generated json files will then be added to the release.

# Prerequisites
Github action needs write permission in order to add the generated files to the release

### CODE REVIEW

**Comment by tschmidtb51** [comment]
The code itself looks OK but before we merge it into the repo: Do you have a release where we can review the results of the action?

**Comment by MarcoKorinth** [comment]
> The code itself looks OK but before we merge it into the repo: Do you have a release where we can review the results of the action?

Here's a test release on the BIZ repo: [Test Release](https://github.com/BIZFactoryGmbH/secvisogram/releases/tag/testrelease)

**Comment by tschmidtb51** [comment]
> > The code itself looks OK but before we merge it into the repo: Do you have a release where we can review the results of the action?
> 
> Here's a test release on the BIZ repo: [Test Release](https://github.com/BIZFactoryGmbH/secvisogram/releases/tag/testrelease)

Thanks. I had a quick look - the version information of secvisogram is missing. Could we add that or check that it is there once we produce it based on tags?

**Comment by MarcoKorinth** [comment]
Adding the secvisogram version to its package.json resolved the issue.
See [Test Release](https://github.com/BIZFactoryGmbH/secvisogram/releases/tag/sbomTest)

File: .github/workflows/sbom.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    name: "CycloneDX SBOM generation"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: anchore/sbom-action@v0
+        with:
+          path: .
+          format: cyclonedx-json
+          artifact-name: sbom.secvisogram_cyclonedx.json
```
Comment (tschmidtb51) [inline .github/workflows/sbom.yml:30]: Similar here:
```suggestion
          artifact-name: sbom.secvisogram.cdx.json

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Review by StefanNemeth — APPROVED** [review]

**Pull request merged**